projects/1.20-fabric/assets/create-deepfried/create_deepfried/lang/zh_cn.json

```
@@ -0,0 +1,48 @@
+{
+  "item_group.create_deepfried.deepfried": "机械动力：油炸",
+  "create_deepfried.group.main": "机械动力：油炸",
+
+  "item.create_deepfried.yuca.tooltip.part1": "获得途径为破坏",
+  "item.create_deepfried.yuca.tooltip.part2": "枯萎的灌木",
+  "item.create_deepfried.yuca.tooltip.part3": "。",
+
+  "item.create_deepfried.european": "§7欧洲 §f\uEFD2",
+  "item.create_deepfried.uk": "§7欧洲 §f\uFDD1",
+  "item.create_deepfried.germany": "§7欧洲 §f\uFDD2",
+  "item.create_deepfried.greece": "§7欧洲 §f\uFDD3",
+  "item.create_deepfried.italy": "§7欧洲 §f\uFDD4",
+  "item.create_deepfried.african": "§7非洲 §f\uDCD2",

```

```suggestion
  "item.create_deepfried.african": "§7非洲 §f\uFDD5",
```
经查验jar里的font/default字体描述文件，推断如此
请检查如此更改后是否在游戏内能正常显示
为bot正常工作，英文文件请一并替换

---

游戏里暂时还没有用到这个条目的物品。不过改了之后倒是可以进入游戏

---

游戏里暂时还没有用到这个条目的物品。不过改了之后倒是可以进入游戏

---

> 游戏里暂时还没有用到这个条目的物品

啊，那应该没什么问题，就这么改吧